### ISSUE TITLE
Let's document! GitHub CLI (gh)

### ISSUE THREAD

**Issue by antoine-amara** [tldr-pages/tldr#4701]
I used the github client (gh) and I consult the tldr documentation and it is already very good, but I think we can improve a little bit this command documentation.

First, they're some little mistake on `gh pr view` and `gh issue view`, we can update the command to correspond to the description.

Moreover, we can create other examples to cover the entire github process from repo creation to release:

* add an example to create a repo.
* add an example to merge a PR.
* add an example to create a release.
* add an example to list all releases.

Finally, If it is ok I can propose a PR to make these changes.

After some discussions , it is better to create sub-pages for the `gh` command:

For those not in the know, tldr-pages is about more than just commands. Those commands that are too expansive to fit on 1 page often have multiple subcommands - which we can document on their own pages. For example, try these:

```
tldr git merge
tldr git add
tldr ip address
```

So, let's document `gh`, we need:

- [x] [`alias`](https://cli.github.com/manual/gh_alias) (#5125)
- [x] [`api`](https://cli.github.com/manual/gh_api) (#5127)
- [x] [`auth`](https://cli.github.com/manual/gh_auth) (#4839)
- [x] [`completion`](https://cli.github.com/manual/gh_completion) (#5134)
- [x] [`config`](https://cli.github.com/manual/gh_config) (#4837)
- [x] [`environment`](https://cli.github.com/manual/gh_help_environment) (#5131)
- [x] [`gist`](https://cli.github.com/manual/gh_gist) (#4898)
- [x] [`help`](https://cli.github.com/manual/gh_help) (#5129)
- [x] [`issue`](https://cli.github.com/manual/gh_issue) (#4836)
- [x] [`pr`](https://cli.github.com/manual/gh_pr) (#4717)
- [x] [`reference`](https://cli.github.com/manual/gh_help_reference) (#5130)
- [x] [`release`](https://cli.github.com/manual/gh_release) (#4773)
- [x] [`repo`](https://cli.github.com/manual/gh_repo) (#4897)
- [x] [`secret`](https://cli.github.com/manual/gh_secret) (#5133)

**Comment by navarroaxel** [comment]
I think you should implement the same structure that the ticket #3953 to create pages for all the `gh` subcommands

**Comment by antoine-amara** [comment]
After some discussions , it is better to create sub-pages for the `gh` command:

For those not in the know, tldr-pages is about more than just commands. Those commands that are too expansive to fit on 1 page often have multiple subcommands - which we can document on their own pages. For example, try these:

```
tldr git merge
tldr git add
tldr ip address
```


So, let's document `gh`, at least we need:

- [ ] `gh auth`
- [ ] `gh config`
- [ ] `gh repo`
- [ ] `gh issue`
- [x] `gh pr`
- [ ] `pr release`
- [ ] `gh gist`

**Comment by sbrl** [comment]
Cool, @antoine-amara! Perhaps you could update the original issue description with that list, to make it easier for people to find?

**Comment by bl-ue** [comment]
There are a few more `gh` subcommands not documented by tldr, and since this issue is relatively recent (Oct. 2020), I decided to edit it and reopen it.

**Comment by sbrl** [comment]
All done!

**Comment by bl-ue** [comment]
Thank you @navarroaxel! 🎉
